### DISCUSSION TITLE
adding support for new AWS services

### DISCUSSION THREAD

**Discussion by srajakumari** [cloud-custodian/cloud-custodian#8818] [AWS] [answered]
Hi,

What is the process for adding support for new AWS services?
I am looking for support for kafkaconnect, EventBridge Scheduler etc.

Thank you,
Raja

**Comment by kapilt** [comment, marked as answer]
there are some getting started docs on contribution here, and most of the extant resources serve as examples for adding new ones.
https://cloudcustodian.io/docs/aws/contribute.html
